src/client.ts

```diff
@@ -251,7 +250,6 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
           this.isClientReady = clientResult.success;
           this.isUserReady = userResult.success;
           const clientAndUserReady = this.isClientReady && this.isUserReady;
```

Comment:
NIT: Off topic: Do you want to use `const clientAndUserReady = this.isReady();`

Reply:
Yeah. I could do that. Great find!